src/main/kotlin/no/nav/familie/ef/sak/repository/BehandlingRepository.kt

```diff
@@ -47,6 +47,17 @@ interface BehandlingRepository : RepositoryInterface<Behandling, UUID>, InsertUp
         LIMIT 1)
     """)
     fun eksistererBehandlingSomIkkeErBlankett(@Param("stonadstype") stønadstype: Stønadstype, personidenter: Set<String>): Boolean
+    @Query("""
+        SELECT b.*, be.id as eksternid_id
+        FROM behandling b
+        JOIN behandling_ekstern be ON b.id = be.behandling_id
+        JOIN fagsak f ON f.id = b.fagsak_id
+        JOIN fagsak_person fp ON b.fagsak_id = fp.fagsak_id
+        WHERE fp.ident IN (:personidenter) AND f.stonadstype = :stonadstype AND b.type NOT IN ('BLANKETT', 'TEKNISK_OPPHØR') AND b.status = 'FERDIGSTILT'
+        ORDER BY b.opprettet_tid DESC
+        LIMIT 1
```

Comment:
Jeg tenker att vi ikke skal ha med de som er `annulert`.
Og så tenker jag att vi skal ha med de som er `teknisk_opphør`. Hvis siste er teknisk opphør, og vi sender teknisk opphør på nytt, så er det ju noe som er feil. Sånn att det burde då bli kontrollert om siste er teknisk opphør, hvis det er slik så skal vi kaste en feil? 
Hvis ikke så kommer vi sende ett nytt opphør til økonomi, basert på dataen fra fra nest siste behandlingen, er det ønskelig? 
Eller er det noe annet case som vi trenger å håndtere? 

Fordelen med å ta med teknisk opphør, er att jeg også trenger den samme queryn i https://github.com/navikt/familie-ef-sak/pull/716 👼 

```suggestion
        SELECT b.*, be.id as eksternid_id
        FROM behandling b
        JOIN behandling_ekstern be ON b.id = be.behandling_id
        JOIN fagsak f ON f.id = b.fagsak_id
        JOIN fagsak_person fp ON b.fagsak_id = fp.fagsak_id
        WHERE fp.ident IN (:personidenter)
         AND f.stonadstype = :stønadstype
         AND b.type != 'BLANKETT'
         AND b.resultat != 'ANNULERT'
         AND b.status = 'FERDIGSTILT'
        ORDER BY b.opprettet_tid DESC
        LIMIT 1
```